roles/ovirt-affinity-groups/README.md

```diff
@@ -0,0 +1,51 @@
+oVirt affinity groups
```

Comment:
We had some docs reviews on roles, so it would be nice if you can follow that patter.
In this case every word should start with upper case letter.